lib/qesapdeployment.pm

```diff
@@ -1273,7 +1273,10 @@ sub qesap_cluster_log_cmds {
 sub qesap_cluster_logs {
     my $provider = get_required_var('PUBLIC_CLOUD_PROVIDER');
     my $inventory = qesap_get_inventory(provider => $provider);
-    if (script_run("test -e $inventory") == 0)
+
+    type_string '', terminate_with => 'ETX';
+    type_string '', terminate_with => 'EOT';
```

Comment:
EOT will send a `ctrl-d` ... it will take you out of the shell session.

Reply:
yes, you are right, I thought to verify the two scenarios with one time run, but it failed, now commented "EOT" line.